routers/posts.py

```
@@ -0,0 +1,44 @@
+)
+
+
+# List, search, sort posts
+@router.get("/posts")
+async def get_posts(query: ListPostsQuery = Depends()):
+    return {"success": "get_posts"}
+
+
+# post new post
+@router.post("/posts")
+async def create_post(post: dict):
```

Ruff (0.14.11)

Unused function argument: `post`

(ARG001)